### ISSUE TITLE
SE|Electricity summation mismatch

### ISSUE THREAD

**Issue by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [pik-piam/remind2#472]
From #467
To replicate/inspect
- check out this commit https://github.com/fbenke-pik/remind2/commit/32630fa92350f1fa3596250894d08e872dd88e46
- get this file
  ```
  utils::download.file(
      'https://rse.pik-potsdam.de/data/example/remind2_test-convGDX2MIF_fulldata.gdx',
      'remind2_test-convGDX2MIF_fulldata.gdx', mode = 'wb')
  ```
- run
  ```
  library(tidyverse)
  devtools::load_all()

  d <- convGDX2MIF('remind2_test-convGDX2MIF_fulldata.gdx')

  attr(d, 'summation_errors') %>% 
      filter('SE|Electricity' == variable)
  ```

**Comment by fschreyer** [comment]
Summation gaps were reduced by #480, but still some inconsistency (up to 1%) remaining for variables describing SE electricity balance.

**Comment by LaviniaBaumstark** [comment]
what ist the status of this issue? Do you plan to fix it for the REMIND 3.3.0 release or are we happy with the reduced differences?
